youtube_dl/extractor/rai.py

```diff
@@ -262,29 +256,33 @@ class RaiPlayPlaylistIE(InfoExtractor):
         'info_dict': {
             'id': 'nondirloalmiocapo',
             'title': 'Non dirlo al mio capo',
-            'description': 'md5:9f3d603b2947c1c7abb098f3b14fac86',
+            'description': 'md5:98ab6b98f7f44c2843fd7d6f045f153b',
         },
         'playlist_mincount': 12,
     }]
 
     def _real_extract(self, url):
         playlist_id = self._match_id(url)
 
-        webpage = self._download_webpage(url, playlist_id)
+        media = self._download_json(
+            '%s.json' % urljoin(RaiBaseIE._BASE_URL, 'programmi/' + playlist_id),
+            playlist_id, 'Downloading program JSON')
+
+        title = media['name']
+        description = media['program_info']['description']
 
-        title = self._html_search_meta(
-            ('programma', 'nomeProgramma'), webpage, 'title')
-        description = unescapeHTML(self._html_search_meta(
-            ('description', 'og:description'), webpage, 'description'))
+        content_sets = [s['id'] for b in media['blocks'] for s in b['sets']]
```

Comment:
https://github.com/ytdl-org/youtube-dl#mandatory-and-optional-metafields

Reply:
Thanks!  I will readd `title`!

Reply:
I have actually double-checked that and all mandatory metafields seems available... Can you please elaborate what's the problem with the quoted diff?

Reply:
> Any field apart from the aforementioned ones are considered optional. That means that extraction should be tolerant to situations when sources for these fields can potentially be unavailable (even if they are always available at the moment) and future-proof in order not to break the extraction of general purpose mandatory fields.

- the playlist title and description are not mandatory.
- extraction should not fail if any of the fields `blocks`, `sets`, and `id` are not available.

Reply:
Thanks for clarifying, I will adjust that!

Reply:
This should be addressed by https://github.com/ytdl-org/youtube-dl/pull/27077/commits/1ac644a2a06e84c116e2c21d21a842251b8aeb25